Description: Comment out example buildbot-worker instance
 Upstream is not interested in this patch because it's more os-specific and the
 init-scripts themselves are contributed so not much connected with buildbot
 itself.
Author: Andriy Senkovych <jolly_roger@itblog.org.ua>
Last-Update: 2011-03-01

--- a/contrib/init-scripts/buildbot-worker.default
+++ b/contrib/init-scripts/buildbot-worker.default
@@ -4,9 +4,9 @@
 # 'true|yes|1' to enable instance and 'false|no|0' to disable. Other
 # values will be considered as syntax error.
 
-WORKER_ENABLED[1]=0                    # 1-enabled, 0-disabled
-WORKER_NAME[1]="buildbot-worker #1"         # short name printed on start/stop
-WORKER_USER[1]="buildbot"              # user to run worker as
-WORKER_BASEDIR[1]=""                   # basedir to worker (absolute path)
-WORKER_OPTIONS[1]=""                   # buildbot options
-WORKER_PREFIXCMD[1]=""                 # prefix command, i.e. nice, linux32, dchroot
+#WORKER_ENABLED[1]=0                    # 1-enabled, 0-disabled
+#WORKER_NAME[1]="buildbot-worker #1"         # short name printed on start/stop
+#WORKER_USER[1]="buildbot"              # user to run worker as
+#WORKER_BASEDIR[1]=""                   # basedir to worker (absolute path)
+#WORKER_OPTIONS[1]=""                   # buildbot options
+#WORKER_PREFIXCMD[1]=""                 # prefix command, i.e. nice, linux32, dchroot
--- a/contrib/init-scripts/buildbot-worker.init.sh
+++ b/contrib/init-scripts/buildbot-worker.init.sh
@@ -21,15 +21,6 @@
 
 # Source buildbot-worker configuration
 [[ -r /etc/default/buildbot-worker ]] && . /etc/default/buildbot-worker
-#[[ -r /etc/sysconfig/buildbot-worker ]] && . /etc/sysconfig/buildbot-worker
-
-# Or define/override the configuration here
-#WORKER_ENABLED[1]=0                    # 0-enabled, other-disabled
-#WORKER_NAME[1]="buildbot-worker #1"         # short name printed on start/stop
-#WORKER_USER[1]="buildbot"              # user to run worker as
-#WORKER_BASEDIR[1]=""                   # basedir to worker (absolute path)
-#WORKER_OPTIONS[1]=""                   # buildbot options
-#WORKER_PREFIXCMD[1]=""                 # prefix command, i.e. nice, linux32, dchroot
 
 
 # Get some LSB-like functions
